.gitignore

```diff
@@ -3,6 +3,9 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+src/Linux/ext/intel/*
+*.o
```

Comment:
Why only exclude .o files in src/Linux/ext/intel and not all .o files (similar to the blanket exemption to .obj files below)? And why only .o files - there are other intermediate files (a.out for instance) on linux that should probably be ignored.